vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -54,41 +54,40 @@ Go to [Identity Management and Access on EKS](https://docs.aws.amazon.com/eks/la
 ### Create the EKS Cluster
 1. Go to `Services` and select `Elastic Kubernetes Service`
 2. Click `Add cluster` → `Create`
-3. Add a cluster name, e.g. (e.g. YOUR-CLUSTER-1).
-4. Select 1.21 for the Kubernetes version.
-5.  For `Cluster Service Role`, select the role you created above, `YOUR-EKS-ROLE`.
-5. Click `Next`.
-6. Use the defaults for `Networking` (Step 2) and click `Next`. 
-7. Use the defaults for `Configure Logging` (Step 3) and click `Next`.
-8. Review your settings and then click `Create`. Cluster provisioning may take several minutes.
+3. Add a cluster name, e.g. `YOUR-CLUSTER-1`
+4. Select 1.21 for the Kubernetes version
+5. For `Cluster Service Role`, select the role you created above, e.g.`YOUR-EKS-ROLE`
+6. Click `Next`
+7. Use the defaults for `Networking` (Step 2) and click `Next` 
+8. Use the defaults for `Configure Logging` (Step 3) and click `Next`
+9. Review your settings and then click `Create`. Cluster provisioning may take several minutes.
 
 See [Creating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) for more detailed information.
 
 ### Add a Node Group to the Cluster
 1. Go to `Services` → `Elastic Kubernetes Service` → `Clusters` and select`YOUR-CLUSTER-NAME` 
 2. Go to `Configuration` → `Compute` 
-3. Click `Add Node Group`
-4. Enter the following:
+3. Click `Add Node Group` and supply the following fields
       * `Name`: Give your group a name, e.g. YOUR-NODE-1
```

Comment:
backtick YOUR-NODE-1

Reply:
@jyunmitch didn't we decide on no backticks for placeholders last fri or am i losing my mind??

Reply:
No you're not losing your mind. My thought was that this should be backticked because it needs to be entered into a field. It makes sense not to backtick when we are simply talking about it, but if it is specifically being entered into a field, it seemed needed. I'm unsure but wondering if we should look at this as all or nothing situation.

Reply:
hmm... we have not been backticking field values that are not somehow already embedded in the application, e.g. values that are user-created or arbitrary. i believe these placeholders fall into that category because they are unique to the user and are not part of the platform or associated scripting. i don't think this is an all or nothing situation, just that *only* terms that are meaningful beyond a specific user implementation should be backticked in order to highlight their significance

Reply:
also, i'm glad i'm not losing my mind. it's touch-and-go on a daily basis.